### PR TITLE
Update LLVM to 19.1.3

### LINKED ISSUE #131031: LLVM regression in beta with `-Ctarget-feature=-simd128`
### Code

I tried this code:

```rust
use std::arch::wasm32::*;

#[target_feature(enable = "simd128")]
pub unsafe fn some_simd128_fn(chunk: v128) -> bool {
    u8x16_all_true(chunk)
}
```

Flags: `-Ctarget-feature=-simd128 --target wasm32-wasip1`

[Godbolt](https://godbolt.org/z/bsa9rzvYP)

I expected to see this happen: compiles

Instead, this happened: fails to compile with "rustc-LLVM ERROR: Do not know how to split this operator's operand!"

### Version it worked on

It most recently worked on: 1.81.0

### Version with regression

`rustc --version --verbose`:
```
rustc 1.82.0-beta.4 (8c27a2ba6 2024-09-21)
binary: rustc
commit-hash: 8c27a2ba6b21f3406a51118643080f0591949827
commit-date: 2024-09-21
host: x86_64-unknown-linux-gnu
release: 1.82.0-beta.4
LLVM version: 19.1.0
Compiler returned: 0
```

```
rustc 1.83.0-nightly (2bd1e894e 2024-09-26)
binary: rustc
commit-hash: 2bd1e894efde3b6be857ad345914a3b1cea51def
commit-date: 2024-09-26
host: x86_64-unknown-linux-gnu
release: 1.83.0-nightly
LLVM version: 19.1.0
Compiler returned: 0
```

@rustbot modify labels: +regression-from-stable-to-beta -regression-untriaged +A-LLVM

### PR DESCRIPTION
**Pull request by dianqk**
Closes #131031.

r? nikic

### CODE REVIEW

**Comment by jieyouxu** [comment]
This doesn't have a regression test because it's a PITA (or next to impossible) to test for #131031 right?

**Comment by DianQK** [comment]
> This doesn't have a regression test because it's a PITA (or next to impossible) to test for #131031 right?

Looks like this is worth adding a test case.

File: tests/ui/target-feature/wasm-disable-simd.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//@ only-wasm32-wasip1
+//@ compile-flags: -Ctarget-feature=-simd128 --crate-type=lib
```
Comment (jieyouxu) [inline tests/ui/target-feature/wasm-disable-simd.rs:2]: Question: does this need `//@ needs-llvm-component: webassembly`?

Comment (DianQK) [reply tests/ui/target-feature/wasm-disable-simd.rs:2]: I don't think it's needed. This is my copy from https://github.com/rust-lang/rust/blob/80b74d397fffc9deaafd2b44552ba3a9107b4d5b/tests/ui/target-feature/wasm-relaxed-simd.rs .

**Comment by Zalathar** [comment]
Not rolling up LLVM updates seems like a good idea in general.

@bors rollup=never

**Pull request merged**